src/show/components/Navbar/index.js

```diff
@@ -193,13 +246,21 @@ const Navbar = () => {
                   />
                 </div>
               </div>
+              <div
+                onClick={() => {
+                  setOpenLogInModal(false);
+                  setOpenSignUpModal(true);
+                }}
```

Comment:
Just create a new function with this, it's ugly, shouldn't be here :(

Reply:
allright